docker_config/dockerfiles/Dockerfile.worker

```diff
@@ -10,6 +10,7 @@ RUN make
 FROM ubuntu:16.04
 ARG default_user=codalab
 MAINTAINER CodaLab Worksheets <codalab.worksheets@gmail.com>
+# the singularity container runtime needs to be able to lookup the uid of the default user
```

Comment:
```suggestion

# The singularity container runtime needs to be able to lookup the uid of the default user
```

Reply:
todo @adiprerepa : figure out if this `useradd` command is needed

Reply:
alright, it is required -- https://www.gitmemory.com/issue/hpcng/singularity/5962/828040364 -- a user needs to be present with a UID

Reply:
I don't understand, could you explain further?

Reply:
@epicfaace updated comment as discussed